### ISSUE TITLE
Frankenstein Variorum Views and Discussion

### ISSUE THREAD

**Issue by ebeshero** [newtfire/introDH-Hub#21]
## Explore the [Frankenstein Variorum Project](https://newtfire.github.io/introDH-Hub/timeMap.html) and related resources and background:
* PSU Behrend campus news article: https://behrend.psu.edu/feature/digital-text-brings-new-life-frankenstein
* Distant view of the whole novel and its 5 versions: https://slides.com/elisabeshero-bondar/frankie#/0/30 
* my slideshow: https://slides.com/elisabeshero-bondar/frankie (This gets a little technical sometimes--but see if you can follow the gist of what we're trying to do.)

## How to read the Variorum Frankenstein: 
* Open the [Variorum Viewer](https://frankensteinvariorum.github.io/viewer/viewer/) to the first fragment available of the 1816 manuscript, and begin exploring that first page. (Here is [the Shelley-Godwin Archive view of that page](http://shelleygodwinarchive.org/sc/oxford/frankenstein/volume/i/#/p1).)
* Then select any of the other versions and begin reading *Frankenstein* **from section 1 through the first 3 or 4 sections for Monday 11/2**. 
* Click on passages highlighted in grey as variants and take note of any particularly interesting alterations that make a substantial change to the way we read a passage. Come prepared to share at least one of these in class discussion.
* Try toggling the various kinds of notes to see the manicule notes view as well.

## What to discuss here (options): Address two or three of the following:
* Locate one or two passages in which you noticed that the versions are quite significantly different from one another. What kinds of serious changes do you see the author making over her text? 
* What kinds of small changes also seem worth noticing, and might affect the way we read?
* Version differences worth noting: The Manuscript Notebook is missing the opening sections and begins late. When that begins: what changes do you think are especially significant from the notebook to the print publication of 1818?
* What do you think Mary Shelley was doing with the Thomas Copy edition? How do these changes compare / contrast with what she altered in the 1831 edition?
* What do you notice about the "user experience" of investigating the Frankenstein Variorum Viewer?  Does the interface help you to make interesting discoveries? Or where does it become distracting or cumbersome? What aspects of the reading interface do you think we should try to improve? Share screen captures that show interesting and/or problematic aspects of the interface.

**Comment by am0eba-byte** [comment]
Reading the 10 sections using the 1818 edition of Frankenstein as the default, I found it fascinating and extremely curious how Mary Shelley changed entire narratives of characters between this edition and the 1831 & Thomas editions. There is one such passage in section 6 that, in the 1818 version, is comprised of only 3 sentences. In that small passage, Victor writes to his sister of how the stranger he had rescued from the arctic began talking to him about his endeavors. All that Mary Shelley includes is that the stranger gives Victor good advice, and Victor perceives him as a very amicable and compassionate person. Meanwhile, in the 1831 edition, this brief description of the characters’ budding connection is super-imposed into four entire lengthy paragraphs, containing new dialogue and a much deeper read into the stranger’s character, his woes, and his deep connection & sympathies with Victor. This lengthy and revealing expansion of the passage from the 1818 and 1823 editions in the 1831 edition adds a whole new layer of intrigue and mystery onto readers’ perspective of the stranger. 

![Screen Shot 2020-10-30 at 10 40 53 AM](https://user-images.githubusercontent.com/70165410/97718550-73813500-1a9c-11eb-9b2f-29c7bca22d69.png)
![Screen Shot 2020-10-30 at 10 43 05 AM](https://user-images.githubusercontent.com/70165410/97718854-c65aec80-1a9c-11eb-8ca5-5a2186ee609b.png)


There were so many other interesting differences between the Thomas edition and the others, but this one from the 1831 edition was undeniably significant: in section 7, before the 6th paragraph in the 1818 edition (beginning with “When my father became a husband and a parent...”), Mary Shelley inserted an extremely thorough background narrative to Victor’s parents, how they met and fell in love, their tender and loving parenting throughout Victor’s childhood, and the story of how his parents found his sister Elizabeth and adopted her from a poor family in Italy – all in the 1831 edition. Omitting that whole background, I suppose you still gather the understanding of the narrator’s deep love he has for his parents and his sister in the editions before 1831, but by adding this touching background into Victor’s parents’ and sister’s pasts, Mary Shelley paints a much more vivid picture of the connections between these characters, and adds a more powerful bond between reader and character, allowing readers to envision these characters and their pasts in much deeper detail – like she did with the passage I mentioned above with the stranger on the boat. 

There are some smaller yet still interesting changes that change the tone and the way information is perceived by the reader – one being a passage in section 3 where Robert is writing to his sister about a seaman on board with him, who once tried to convince his ex’s father to let her be with her desired lover instead. Robert tells his sister in the letter that while the man’s actions were noble, he isn’t educated much, and doesn’t have many inclinations beyond being a sailor. In the 1818, 1823, and Thomas editions, all that is written is that he “doesn’t have much of an idea beyond the rope and shroud,” while in the 1831 edition, he writes instead that he is “wholly uneducated,” and his air of ignorant carelessness sort of detracts from his capacity to be fully sympathized with. This change, while somewhat minor, totally changes the way the reader perceives Robert’s perception of the sailor. The 1831 edition sort of reflects a different attitude towards the sailor within Robert, while the other editions seem to mention the fact of his lack of education in mere passing. The tone of this detail shifts quite a bit between the two versions – and I’ve noticed many other instances where different editions reflect minor tone changes like this as well.  

It seems, from the Thomas Copy edition to the 1831 edition, many of the strike-throughs, insertions, deletions, and substitutions that Mary Shelley made by hand in the Thomas copy were reflected almost exactly as corrections in her 1831 edition.  You can especially see evidence of this in section 7, where in all of the other editions the passage in the screenshot below is included in some form – in the Thomas copy, she crossed the whole thing out – and then in the 1831 edition, the passage she struck through has been omitted. 

![Screen Shot 2020-10-30 at 10 29 09 AM](https://user-images.githubusercontent.com/70165410/97718269-22714100-1a9c-11eb-9de1-f53c7d3f29db.png)

**Comment by arrowarchive** [comment]
I've noticed some interesting details when reading _Frankenstein_ over Variorum. I used the Thomas edition of the text as the default, and I made comparisons from there.

The first thing I noticed was that every volume besides the manuscript had a letter narrative (pictured below). This can be attributed to a lack of information prior to section seven, but it is still noteworthy.
![Screenshot (115)](https://user-images.githubusercontent.com/70866633/97806271-d99ac300-1c28-11eb-9e7d-bbbcd35260af.png)

There's also a number of differences between the Thomas edition and the 1831 edition. In particular, the spelling. There are several examples (pictured below) which feature spelling differences between the two volumes. The 1831 edition uses the modern spelling of several words, such as "wrapped" and "Shakespeare", and the punctuation for several words is also different (picture 3). The 1831 edition and the Thomas edition have the most striking differences, as they are similar to the 1818 and 1823 editions otherwise.
![Screenshot (112)](https://user-images.githubusercontent.com/70866633/97806405-7d846e80-1c29-11eb-8dc5-58a48e479d5c.png)
![Screenshot (111)](https://user-images.githubusercontent.com/70866633/97806440-a9075900-1c29-11eb-8b72-5ca4318a623b.png)
![Screenshot (120)](https://user-images.githubusercontent.com/70866633/97806611-8e81af80-1c2a-11eb-8da6-3682b42f65b0.png)

My last note for the comparison is that besides the spelling, there are several differences in length between the two drafts. Noticeably, whenever the Thomas edition has a longer summary, the other versions have shorter summaries, and the opposite is true for the 1831 version. It emphasizes their differences, while acknowledging the similarities between the 1818 and 1823 versions.

![Screenshot (114)](https://user-images.githubusercontent.com/70866633/97806851-ba516500-1c2b-11eb-81a4-21008eb015dc.png)
![Screenshot (118)](https://user-images.githubusercontent.com/70866633/97806854-bcb3bf00-1c2b-11eb-9966-2a3e9351aba7.png)


When working with Variorum, there are times I found the interface cumbersome. I'd search for differences, only to find some areas that were unusually highlighted (such as the letter introductions) or areas that highlighted so much that it was difficult knowing where to begin. In my opinion, the interface for the website is cluttered, and I feel like it will need to be organized when examining future texts.

**Comment by nxh5137** [comment]
I used the 1831 version of _Frankenstein_ for the comparison and noticed some differences from the other versions of the text.

In the first picture below, in section 6, the 1831 version is much longer than the versions that came before it, and the manuscript version is not present. After reading both the comparisons, I noticed that they are both different from each other, and I am not just talking about the length. They also seem to be talking about two different things almost.

![Screenshot 2020-11-01 at 9 10 35 PM](https://user-images.githubusercontent.com/34407859/97873799-1f699100-1ce6-11eb-841b-a72d8bda1dfe.png)

 Some other changes in the 1831 version in section 6 were small for the most part. For example, in most versions before this, it says "once," while in the 1831 version, it decides to put in more words by saying "at one time,". A small change I did notice that makes it seem different. In the previous versions, it said "laugh", but in the 1831 version, it says "smile" instead. while small, I think that it says two different things when they replaced the word.

![Screenshot 2020-11-02 at 8 52 47 AM](https://user-images.githubusercontent.com/34407859/97888095-bb03fd00-1cf8-11eb-95e8-792b38be4a4f.png)

What I noticed about the Variorum Viewer is that it gets multiple editions of something and then compares that to the other editions. I did make some interesting discoveries on how each text differs both big and small. While I am not sure how to improve upon the interface,  a suggestion I have would be to select which editions we would like to compare instead of all of them at once so the dots aren't everywhere

**Comment by kzp308** [comment]
I chose to look at the similarities and differences in the 1831 version and the Thomas copy version. When I looked through the 1831 version of Frankenstein, I found that there was a lot of better editing choices. I prefer the 1831 version to the Thomas copy version for this reason. The Thomas version was not bad with the editing, but it definitely felt more like the rough draft of a final paper. There were some cuts taken out and punctuation added in the 1831 version. In the Thomas screenshot, the word "Tempest" is put together with "and". In the 1831 screenshot, the words have been correctly separated. 
![Screenshot (79)](https://user-images.githubusercontent.com/70867065/97900438-9fa0ee00-1d08-11eb-89c6-b6cbc49ab5b6.png)
![Screenshot (80)](https://user-images.githubusercontent.com/70867065/97900448-a465a200-1d08-11eb-8439-18f20cfa51ea.png)
Then, I noticed that there was a lot of crossing out of certain words. When I saw them crossed out in the Thomas version I initially thought that they wouldn't be in the 1831 version. The 1831 version seemed to be more edited, but in the screenshots below the words were chosen to stay in the text so, I'm not quite sure why she crossed them out originally. Maybe she changed her mind halfway through?
![Screenshot (81)](https://user-images.githubusercontent.com/70867065/97900750-1d64f980-1d09-11eb-80c5-ad72077a3984.png)
![Screenshot (82)](https://user-images.githubusercontent.com/70867065/97900758-20f88080-1d09-11eb-8ed2-9e7222b623dd.png)
Lastly, I noticed a lot of editing changes that would change the way I read the text. In section 2, "sea-faring" or "seafaring" would definitely change the pronunciation I would use on the word. 
![Screenshot (83)](https://user-images.githubusercontent.com/70867065/97900981-8187bd80-1d09-11eb-905b-4f381f743180.png)

**Comment by amw6765** [comment]
Throughout reading the different versions of this story there were a lot of subtle differences regarding the use of words and sentence structures. For example, in section 7 when talking about his brothers the manuscript version sees to have a few less words than that compared to the 1818, 1823, and the Thomas versions. After reading more I began to see that the manuscript version was shorter in how the sentences were written. I can attribute to this because the newer versions most likely were adding more to the story to lengthen It and be more descriptive. 
![Screen Shot 2020-11-02 at 1 32 11 PM](https://user-images.githubusercontent.com/70722523/97905360-d4fd0a00-1d0f-11eb-8e53-dcf02cd0bceb.png)

After working with the interface for a while I came to a conclusion. I believe that the Viewer was able to make me more clearly see the subtle and large differences between the 5 different versions. Because of the color coded dots that are underneath the lines of text it can quickly allow you to see what has changed or remained the same. 
![Screen Shot 2020-11-02 at 1 36 03 PM](https://user-images.githubusercontent.com/70722523/97905670-63718b80-1d10-11eb-90f5-3c01953739d9.png)
![Screen Shot 2020-11-02 at 1 36 07 PM](https://user-images.githubusercontent.com/70722523/97905678-653b4f00-1d10-11eb-8177-a2b5b2567cf0.png)
However, at first it was a bit confusing to me because I did not yet understand why certain things had lines striking through them or why certain things were underlined and others were not. After some time I was able to figure it out. But I believe that this viewer could be improved a bit by maybe having a key/legend that will go over an example sentence. I feel that if the viewers could look at an example with a description, they would be able to more quickly see why the Viewer behaves in the manner we see. I believe that was the biggest issue that I encountered while working with this. Other than that I believe that the simplicity of this Viewer is great!

**Comment by bpm5520** [comment]
When reading through section 6 in the 1831 edition, I noticed a few key differences compared to other versions. The main one being that the manuscript does not have this section in it, though the manuscript doesn't have 6 of the 10 sections that are on this variorum. As well, a few times in the section, it showed that the Thomas version had more detailed sentences compared to the 1831 edition, and the 1831 edition had more fleshed out sentences than other editions. 
![Variorum1](https://user-images.githubusercontent.com/70908691/97909529-5f486c80-1d16-11eb-83bd-af0014f5204e.JPG)
![Variorum2](https://user-images.githubusercontent.com/70908691/97909530-5fe10300-1d16-11eb-88cb-1bb5f643b6d5.JPG)

The viewer is very good for helping to spot differences between the sections instead of reading each one together to spot them. One thing of note that would improve this setup is to have the ability to choose 2 or more of the pages to compare together, instead of all 5 at once, tossing dots everywhere. The highlighting and use of the dots to spot differences is a major help though.

![Variorum3](https://user-images.githubusercontent.com/70908691/97910130-4b513a80-1d17-11eb-94e8-27497f6c0933.JPG)

**Comment by tomsheehy** [comment]
To start, I found the interface simple and easy to navigate. I was never overwhelmed by information which made analyzing the different passages fun and easy as well. I compared the 1823 version to the rest of the versions and I was quite fascinated. While reading, I found that the 1823 version seems to have less punctuation and detail than a few of the others. 
<img width="1680" alt="Screen Shot 2020-11-02 at 2 34 07 PM" src="https://user-images.githubusercontent.com/70911575/97911622-8e141200-1d19-11eb-8f45-c7d3efd46017.png">
As seen above, the 1823 version includes much less information in just one paragraph of the passage. 
<img width="1680" alt="Screen Shot 2020-11-02 at 2 42 36 PM" src="https://user-images.githubusercontent.com/70911575/97911675-ad12a400-1d19-11eb-8be2-9b3188f0fc48.png">
Above, is another good example of the 1823 version having less detail. In 3 versions, the sentence is ended with "safety." while the 1831 version includes more detail.
Some of the small changes between the passages really bothered me. For example, we can see in Section 5 the slight changes between the passages.
<img width="1680" alt="Screen Shot 2020-11-02 at 2 47 02 PM" src="https://user-images.githubusercontent.com/70911575/97912119-5d80a800-1d1a-11eb-920a-11d2d78de4d8.png">
I was only bothered because I would personally never have the character to change such specific things. After I continued to analyze these changes more I appreciated the way it makes me read the passages.

**Comment by AlexanderRAnderson** [comment]
I was comparing general differences between all the versions of the text and something jumped out at me immediately. The revisions of the original text seem to make the language more concise and clear, generally shorter than the original. On the other hand, the 1831
 revision of the text seemed to do the opposite. The message is still more clear than the original in my opinion, but it is much longer and drawn out.
![Frankenstein](https://user-images.githubusercontent.com/61561118/97911816-bcd1be80-1d00-11eb-8bbd-461b8d5625f9.PNG)
![Frankenstein2](https://user-images.githubusercontent.com/61561118/97911839-c78c5380-1d00-11eb-8ced-aefc2c832f36.PNG)
![Frankenstein3](https://user-images.githubusercontent.com/61561118/97911848-ca874400-1d00-11eb-8715-252fe6a1fe94.PNG)

The ideas seem to be more fully formed in the latest revision even if they are longer. It indicates the overall confidence that the revision was made with. It had an idea that it was striving for over just revision of the text.

What do you notice about the "user experience" of investigating the Frankenstein Variorum Viewer? I think the user experience of investigating the Frankenstein Variorum Viewer is fine once you get used to it. At first it is a little difficult to understand what is going on. Once you understand what you are looking at it becomes much easier to use. I think it does help make interesting discoveries about the text. I think it is easier to make general discoveries of style rather than more specific discoveries such as ideas or description choices. I'm not sure what to change about the interface to make it better. I think it does what it wants to do well. The only thing I would change is possibly having a constant legend on the left side of the text rather than one at the top of the screen.
![Variorum](https://user-images.githubusercontent.com/61561118/97912811-2c947900-1d02-11eb-95f0-0e0b0fce9314.PNG)

**Comment by mws6176** [comment]
The section 7 of the 1831 edition of _Frankenstein_ exemplifies many errors throughout the passage, as apposed to the 10th version which has fewer corrections. The topics of each are also varied. Here is part of the 7th section: 
![Screen Shot 2020-11-02 at 2 52 53 PM](https://user-images.githubusercontent.com/70854997/97912634-10510600-1d1b-11eb-82eb-33a6c0ee7114.png)

It can be seen how there are clearly more markings on this one rather than the 10th section seen here: 
![Screen Shot 2020-11-02 at 2 55 31 PM](https://user-images.githubusercontent.com/70854997/97912857-6c1b8f00-1d1b-11eb-9cf4-a3c301ab68e3.png)

The majority of the themes and motifs from the 7th section are much different than those of the 10th. The 10th seems to be much more thought out and thorough whereas the 7th is somewhat scattered and inconsistent.

**Comment by BenS6206** [comment]
I would say it was very interesting to see the changes to sections over time, with some sections going over changes with each and every iteration. 
![Changes](https://user-images.githubusercontent.com/69773455/97914491-f664f280-1d1d-11eb-9404-725e3c3a78c0.png)
on the MS version it was interesting to see the crossed out areas that were corrected in itself. In the image above it can be seen that the Thomas version completely removed the selected section, while 1818 and 1823 used a more cleaned up version of the section. It is worth noting that even though the later sections.

Side note, something else minor that I found somewhat interesting was the difference in the spelling of the name "Shakespeare" through different renditions as the 1831 version lists his name as "Shakspeare", which I don't believe to be a typo as on wikipedia it says that his name was often spelled this was in the early 19th century.  
![1831-Shakspeare](https://user-images.githubusercontent.com/69773455/97915299-42fcfd80-1d1f-11eb-99db-7fd3246e0924.png)

**Comment by natalyamyers** [comment]
The first section I noticed that had varied differences was the beginning of chapter 2. 
<img width="699" alt="1" src="https://user-images.githubusercontent.com/70730578/97914335-b69e0b00-1d1d-11eb-8f42-155dd5e36b4b.PNG">
I find it really interesting that she chose to take a different approach to this opening. I think I like the other versions as opposed to the original MS, just because I think the other versions invoke more imagery in your mind and seems more poetic in a way.

Another extremely varies section was farther down in Chapter 2, where the 1831 version added a much larger section that none of the other versions had.
<img width="677" alt="5" src="https://user-images.githubusercontent.com/70730578/97915011-c4a05b80-1d1e-11eb-9d9e-2fac3b0c01d6.PNG">
<img width="194" alt="2" src="https://user-images.githubusercontent.com/70730578/97915014-c538f200-1d1e-11eb-912e-ce448cc28e71.PNG">
<img width="194" alt="3" src="https://user-images.githubusercontent.com/70730578/97915016-c538f200-1d1e-11eb-81a4-71c853f8e67f.PNG">
<img width="193" alt="4" src="https://user-images.githubusercontent.com/70730578/97915018-c538f200-1d1e-11eb-9189-dcb49efaee54.PNG">
I think the longer addition adds some better depth to the situation than other versions did with only a few sentences. In the 1831 version as well, I feel that the mention of electricity and the construction of souls is a nice foreshadow to the creation of Frankenstein's monster later in the novel.

In the vein of user friendliness, I noticed a few things that would have been nice to have had. First, i didn't like that you had to click on the dots underneath a grey highlighted block of text to open up the versions, it would have been nice to be able to just click on the text block you wanted to open. Second, I kept clicking on blocks to open and then immediately losing where in the text they came from. An update where you could see the section you're viewing is highlighted in yellow or had something else to denote it as the selected section.

**Comment by pxl5083** [comment]
Reading through the different issues and looking at the differences, one thing is consistent and that is the fact that the Thomas edition happens to be more descriptive and quick to the point. 

![Screenshot (24)](https://user-images.githubusercontent.com/70915449/97915914-40e76e80-1d20-11eb-9b39-5201d2a52259.png) 

Another instance of this: 
![Screenshot (25)](https://user-images.githubusercontent.com/70915449/97916092-7ee49280-1d20-11eb-9f15-7c10765d154e.png)

To build of this, I feel as thought all the other editions have significantly more in common and are more alike than compared to any of the other editions when put up next to the thomas edition.

This program took me a little while to figure out but overall it was not overly difficult to operate. I think that instead of having to click to gain more information, you should be able to view it just on the page rather than having to open up a separate pop-up.

**Comment by NickyV1234** [comment]
![frank1](https://user-images.githubusercontent.com/70916831/97930087-acd5d100-1d38-11eb-9757-9f2b037e647f.JPG)
![frank 2](https://user-images.githubusercontent.com/70916831/97930094-af382b00-1d38-11eb-9955-fb701349c3b7.JPG)
![frank 4](https://user-images.githubusercontent.com/70916831/97930102-b0695800-1d38-11eb-82d5-719a05628af9.JPG)
![frank 3](https://user-images.githubusercontent.com/70916831/97930110-b2331b80-1d38-11eb-836e-2af2c0a467cf.JPG)
one thing i noticed is that there are very few annotations where all three versions are different. most of the highlights involve a disagreement between two versions. one of the images show that some of the versions dont really go into much emphasis on what is happening in the background or what the environment is doing. there are other screenshots that show one version going into the description of steeples where as another version talks about who the steeples belong to.

**Comment by xsierraallen** [comment]
I chose to read the Thomas Edition of Frankenstein as my default when comparing works. When looking at the first section, I compared it to the 1823 edition. The two works were extremely similar except for a few spelling errors here and there. 
![Screenshot (23)](https://user-images.githubusercontent.com/70916885/97933290-4f458280-1d40-11eb-9f85-cb26e3add432.png)
I decided to compare the Thomas editions to the MS edition and I found much better results. The MS edition made multiple changes throughout the text removing unneeded words, basically the entire text was marked up. This created a whole different text to compare to the Thomas text which was very interesting.
![Screenshot (24)](https://user-images.githubusercontent.com/70916885/97933378-96337800-1d40-11eb-995b-23d0c09949c5.png)
![Screenshot (25)](https://user-images.githubusercontent.com/70916885/97933379-96cc0e80-1d40-11eb-9298-2387e39c29b3.png)
This program was very interesting to use. I enjoyed it but it was a little tedious that the annotations had to be clicked on to view, but not everyone wants them out in the open which is reasonable. It was interesting to see how each work did something a little different to the text whether they tweaked it a bit or completely removed a piece or even added something of their own.

**Comment by hjl5363** [comment]
I decided to answer the first two questions. Personally, I liked the 1818 version better due to the fact that too many things on your screen and excessive highlighting can be bothersome. The MS version is much worse with highlighting compared to the 1818 version. I think this is directly
<img width="1791" alt="Screen Shot 2020-11-02 at 8 30 39 PM" src="https://user-images.githubusercontent.com/70916576/97936782-a0f30a80-1d4a-11eb-9e1a-16077822cfb3.png">
<img width="1791" alt="Screen Shot 2020-11-02 at 8 30 58 PM" src="https://user-images.githubusercontent.com/70916576/97936783-a0f30a80-1d4a-11eb-92b7-a5235935b927.png">

 affecting how we read.

**Comment by gabbiedoster** [comment]
Before starting this assignment I briefly skimmed over each edition of the texts to see which one would catch my interest. For my comparisons, I decided to use the text analyze the text from the Thomas Edition and the 1832 edition of Frankenstein! I found these texts, personally, the most interesting way of telling the story. Secondly, I found many similarities, but found some interesting differences between the two.

<img width="853" alt="Thomas Edition S2" src="https://user-images.githubusercontent.com/70911505/98056597-47ecaa80-1e0e-11eb-8a81-4840202d30c1.png">
  ---------------------------------------------------------------------------------------------------------------------
<img width="813" alt="1823 Edition S2" src="https://user-images.githubusercontent.com/70911505/98056603-4d49f500-1e0e-11eb-9607-8fce1b1ff2bc.png">

For my first comparison, I noticed how the text found in section two of the Thomas Edition (top image) felt more structured and detailed compared to section two of the 1823 Edition (bottom image). The lines, "What may not be expected in a country of eternal light?ruled by different laws and in which numerous circumstances enforce a belief that the aspect of nature differs essentially from anything of which we have any experience", were found in the Thomas Edition and not found in the 1823 Edition. I felt although this line was ultimately extra and the story, such as in the 1823 Edition did well without it. In the 1823 Edition,  I felt although the text was more direct and got to the point. The text felt more direct and I found it less confusing reading it versus the Thomas Edition text. I think there isn't much wrong with including more descriptive details like in the Thomas version, but I personally found the 1823 version to have more flow and intimacy to the story and characters. 

<img width="853" alt="Thomas Edition S2" src="https://user-images.githubusercontent.com/70911505/98057134-8c2c7a80-1e0f-11eb-94b4-84589b7fa684.png">
  ---------------------------------------------------------------------------------------------------------------------
<img width="891" alt="1823 Edition S4" src="https://user-images.githubusercontent.com/70911505/98057144-9189c500-1e0f-11eb-8cf1-1c657aeb0250.png">

In this comparison of the Thomas Edition (top image) and the 1823 Edition (bottom image) I found extra words and an added sentence in the Thomas Edition. I found the added sentence to help the story become more imaginative and helps the reader to visually indulge in the story. Personally, I think it added a nice visual moment and is beautifully written. Again, like the last comparison I made, I find the 1823 Edition to be very direct and in this comparison in the fourth section of the texts. The extra sentence found in the Thomas Edition isn't included which makes the text feel more intentional and less visual.

**Comment by ebeshero** [comment]
If you'd like to link directly to a specific comment in this issue thread, click the three dots to the top right of it, and choose "Copy link". here's a sample of such a link: https://github.com/newtfire/introDH-Hub/issues/21#issuecomment-720653620
